### ISSUE TITLE
Component pattern

### ISSUE THREAD

**Issue by skeie** [planningcenter/react-patterns#25]
Hi, 

I like alot of the principles here, but, why is the render method at the bottom of the class, in my experience i always look for the render method when I inspect a React class, so why don't put it under the constructor so it will be at the top of the file?

## Component Organization

* class definition
  * constructor
    * event handlers
  * 'component' lifecycle events
  * getters
  * render
* defaultProps
* proptypes

**Comment by chantastic** [comment]
Hi @skeie,

This is mostly preference. I come from a pretty "Crockfordian" style of writing JavaScript—manually hoisting variables and putting the meat of your function at the bottom. While that has no baring on the organization of a class definition, it's a major influencer.

While we compiled this before we'd found any other suitable guides, we've updated it as we've seen people Rally around particular patterns. "`render` at the bottom" seems like the common approach. AirBNB—who's [JavaScript styleguide](https://github.com/airbnb/javascript) has 24k stars—also recommends `render` at the bottom.

All that to say "popular opinion" is the only reason we write `render` at the bottom :)
